docs/windows/latest-supported-vc-redist.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Latest Supported Visual C++ Redistributable Downloads"
+description: "This article lists the download links for the latest versions of Visual C++ Redistributable packages."
+ms.date: 09/09/2021
+helpviewer_keywords:
+  [
+    "redist",
+    "vcredist",
+    "Visual [C++] redistributable",
+    "Visual [C++] downloads",
+    "MSVC downloads",
+    "[C++] downloads",
+  ]
+ms.assetid: 3017a6b3-67d8-465f-9b23-ed0d54559478
+---
+
+# Microsoft Visual C++ Redistributable Latest Supported Downloads
+
+> [!NOTE]
+> Some of the downloads that are mentioned in this article are currently available on [My.VisualStudio.com](https://my.visualstudio.com/). Make sure to log in by using a Visual Studio Subscription account so that you can access the download links.
+> If you are asked for credentials, use your existing Visual Studio subscription account or create a free account by selecting ["Create a new Microsoft account."](https://login.microsoftonline.com/common/oauth2/authorize?client_id=499b84ac-1321-427f-aa17-267ca6975798&site_id=501454&response_mode=form_post&response_type=code+id_token&redirect_uri=https%3A%2F%2Fapp.vssps.visualstudio.com%2F_signedin&nonce=95942429-1297-4a7b-ab5c-0d6fcce90df4&state=realm%3Dmy.visualstudio.com%26reply_to%3Dhttps%253A%252F%252Fmy.visualstudio.com%252FDownloads%253Fpid%253D2082%26ht%3D3%26nonce%3D95942429-1297-4a7b-ab5c-0d6fcce90df4&resource=https%3A%2F%2Fmanagement.core.windows.net%2F&cid=95942429-1297-4a7b-ab5c-0d6fcce90df4&wsucxt=1)
+
+The Microsoft Visual C++ Redistributable installs the Microsoft C++ libraries. If your application uses those libraries, a Microsoft Visual C++ Redistributable package matching your application's architecture and version (or higher) must be installed on the target system before you install your application.
+
+For details on how to install and redistribute Visual Studio components, please see [Redistributing Visual C++ Files](redistributing-visual-cpp-files.md)
+
+## Visual Studio 2015, 2017, and 2019
+
+Below are the latest supported en-US Microsoft Visual C++ Redistributable packages for Visual Studio 2015-2022. The latest supported version has the most recent implemented C++ features, security and reliability bug fixes as well performance improvements. We recommend installing this version for all applications created using Visual Studio 2015, 2017, 2019 or 2022
```

Comment:
"as well performance" -> "as well as performance"

Reply:
Acrolinx doesn't like "as well as". I'd recommend splitting the sentence up, instead:
`The latest supported version has the most recent security, reliability, and performance improvements, It also includes the latest C++ standard language and library conformance changes.`

The text mentions 2022, but the heading doesn't. I thought the 2022 redist was going to be an ABI breaker, so you'd need a separate one. Is that not the case?

Reply:
>> I'd recommend splitting the sentence up.

Updated the text to: `The latest supported version has the most recent implemented C++ features, security, reliability and performance improvements. It also includes the latest C++ standard language and library standards conformance updates.`

>> The text mentions 2022, but the heading doesn't. I thought the 2022 redist was going to be an ABI breaker, so you'd need a separate one. Is that not the case?

No, the 2022 won't be an ABI-breaking. It will officially named _"Visual C++ Redistributable for Visual Studio 2015-2022"_